cypress/integration/event-signup-flow.spec.js

```diff
@@ -717,4 +717,94 @@ describe("Verify page load " + new Date(), () => {
 				});
 			});
 	});
+	it('It nativates to selected "Online Events" page', function () {
+		cy.visit("/events/category/online-events", {
```

Comment:
Can we extract all the `cy.visit` calls into a function that does the auth so its not repeated every time? not just here but I think there's a bunch of tests that would benefit from it

Reply:
Resolved